.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Unneeded empty line. Better start with `---` on the first line as this is a yaml file.